DylibBinder/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using SwiftReflector.Inventory;
+using SwiftReflector;
+using System.IO;
+
+namespace DylibBinder {
+	class Program {
+		static void Main (string [] args)
+		{
+			DylibBinderOptions options = new DylibBinderOptions ();
+			var extra = options.optionsSet.Parse (args);
+
+			if (options.PrintHelp) {
+				options.PrintUsage (Console.Out);
+				return;
+			}
+
+			if (string.IsNullOrEmpty (options.DylibPath)) {
+				Console.WriteLine ("The path to the Dylib was not included. Try using --dylibPath.");
+				return;
```

Comment:
Maybe return a non-zero exit code if there are failures:

```suggestion
				return 1;
```

this also requires changing the function to return `int`, and update the other return statements.

Reply:
Should I add somewhere that checks this return value? Or is this for the future if anything else calls into my Main?

Reply:
This change means that whenever someone uses this executable from scripts or the command line, the executable will return a non-zero exit code (and thus indicate failure), so that scripts can realize that something went wrong.

In other words: there's no need to add checks anywhere.